cs251tk/toolkit/__main__.py

```diff
@@ -72,6 +72,22 @@ def main():
     if date:
         logging.debug('Checking out {}'.format(date))
 
+    try:
+        with chdir(os.path.join(basedir, 'data')):
+            run(['git', 'pull', 'origin', 'master'])
+
+            try:
+                with chdir(os.path.join(basedir, 'data', 'source')):
+                    run('make')
+            except FileNotFoundError:
+                if not quiet or not no_update:
+                    print("Optional add-on program CheckDates not installed.\n"
```

Comment:
Eventually I'd like to move this program into this python script, but that's not a blocker for merging.